### ISSUE TITLE
For `Progress indicator` component, indeterminate state is not read by Voice Over

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1698]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 17

### Device OS version

iOS 26.5.1

### Is Liquid Glass disabled

- [ ] Yes, Liquid Glass is disabled (or OS version is older than 26)
- [x] No, Liquid Glass is enabled (or OS version is 26 or newer)

### OUDS iOS library version

3.0.0

### Describe the issue

In case of indeterminate state, the component is not reached by Voice Over

### Expected Behavior

Progress indicator should be read by Voice Over to inform the user that there's something ongoing, like "Download in progress"

### Device settings

Voice Over

**Comment by pylapp** [comment]
@pya35  Dans la mesure où le composant n'est jamais utilisé seul et qu'il peut avoir un _helper text_ (que l'on a oublié pour le _circular_), quelle serait la meilleure façon de faire ?

**Comment by pya35** [comment]
@pylapp selon les cas, le helper text peut ne pas suffire à comprendre le contexte, un nom générique du genre "barre de progression / barre d'avancement" permettrait de faciliter la chose. 
C'est d'ailleurs valable aussi pour l'état "determinate" contrairement à ce que j'ai indiqué, car le pourcentage entendu par défaut n'est pas assez explicite.

**Comment by pya35** [comment]
@pylapp l'état "indeterminate" pourrait être complété par "Barre de progression **en cours**" (iso Android)

**Comment by pya35** [comment]
@pylapp 2 choses à prévoir : 
- le nom du composant : barre de progression, d'avancement... à customiser selon le contexte
- l'état (description) :  par exemple "téléchargement en cours,"... à customiser selon le contexte

**Comment by pylapp** [comment]
@pya35 Le soucis c'est que l'on ne connaît pas le contexte en fait.

Du coup, est-ce que l'on peut imaginer laisser un champs dédié à ce sujet aux utilisateurs de sorte à ce qu'ils définissent eux-même l'information supplémentaire à vocaliser ?

**Comment by pya35** [comment]
@pylapp complètement, c'est pour ça que je précisais "à customiser", vous pouvez mettre un exemple pour l'app de démo mais modifiable

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **3.0.0**
- Build number: **1788815142**
- Build tag: **d979f56**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **3.0.0**
- Build number: **1788989831**
- Build tag: **5e2d2a8**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **3.0.0**
- Build number: **1789074302**
- Build tag: **bb28c6d**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New release available** 🚀

Please, get the [new release from GitHub](https://github.com/Orange-OpenSource/ouds-ios-design-system-toolbox/releases/tag/3.0.0) and wait for App Store update to test it!

- Display name: **Design System Toolbox**
- Version: **3.0.0**
- Build number: **1789078720**



_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
